### PR TITLE
Added a keyboardInteractive prop for navigating slides with arrow keys.

### PR DESCRIPTION
**Pull request by 7imbrook**
See issue #1 

No way to add customization of keys, thats another feature.

### CODE REVIEW

File: src/torque-slides.jsx [hunk 1]
```diff
@@ -32,6 +40,23 @@ var TorqueSlides = React.createClass({
     this.setInterval(this.updateSlide, 1000);
   },
 
+  keydown(e) {
+    switch (e.keyCode) {
+      case RIGHT_ARROW:
+        this.setState({
+          current: (this.state.current + 1) % this.props.children.length,
+          countdown: this.props.duration
```
Comment (kristenmills) [inline src/torque-slides.jsx]: I think this needs to take into account the possibility of the next slide having a custom duration.  Look at the updateSlide function.

File: src/torque-slides.jsx [hunk 2]
```diff
@@ -32,6 +40,23 @@ var TorqueSlides = React.createClass({
...
+        });
+        break;
+      case LEFT_ARROW:
+        this.setState({
+          current: this.state.current - 1 < 0 ? this.props.children.length - 1: this.state.current - 1,
+          countdown: this.props.duration
```
Comment (kristenmills) [inline src/torque-slides.jsx]: This one too

**Review by kristenmills — CHANGES_REQUESTED** [review]

File: src/torque-slides.jsx (hunk 1 above)
Comment (kristenmills) [reply src/torque-slides.jsx]: Also the onVisible callback. You might want to pull out a lot of the logic of update slide into a separate function and then call it in keydown and update slide.

**Comment by 7imbrook** [comment]
My though behind not doing that (other then complexity) was when you navigate the slides the default would be used, so it will stay longer on a slide when your jumping around. I know that the short slides are only ever used for the black/white slides on the GTV but... you now have a flow that can keep you on a slide longer.

Thoughts?

**Comment by kristenmills** [comment]
You need to at least call on visible. That's how we pull new events and cycle through them for the single event slide. And if you actually want to do an actual keyboard mode, turn off setInterval and the countdown completely and only use the keyboard to control it.

**Comment by 7imbrook** [comment]
@kristenmills review pls

**Review by kristenmills — APPROVED** [review]

**Pull request merged**